docs/conf.py

```diff
@@ -18,8 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Neptune client'
-copyright = '2019, team neptune.ml'
```

Comment:
I believe that this year should not be modified as well

Reply:
commited changes